Makefile

```diff
@@ -57,10 +57,10 @@ DOCKER_RUN_DOCKER := $(DOCKER_FLAGS) "$(DOCKER_IMAGE)"
 
 default: binary
 
-all: build
+all: build ## validate all checks, build linux binaries, run all tests, cross build non-linux binariesand generate archives
```

Comment:
can we invent something to make it wrap, so that it fits in smaller windows (80-chars?)